app.rb

```diff
@@ -91,7 +91,7 @@ def create_rental
 
     date_part = date.strftime('%Y/%m/%d')
     puts "Date: #{date_part}"
-    puts "Rental created successfully"
+    puts 'Rental created successfully'
   end
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍